File: 1160.py
for i in range(0, int(input())):
    A, B, perA, perB = input().split(' ')
    A = int(A) 
    B = int(B)
    perA = float(perA)
    perB = float(perB)
    anos = 0
    while True:
        A += int(A*perA/100)
        B += int(B*perB/100)
        anos += 1
        if anos > 100:
            print(f'Mais de 1 seculo.')
            break
        if A > B:
            print(f'{anos} anos.')
            break

File: 1160.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');
let n = parseInt(lines.shift());
for(var i = 0; i < n; i++){
    let [A, B, perA, perB] = lines.shift().split(" ");
    A = parseInt(A); B = parseInt(B); perA = parseFloat(perA); perB = parseFloat(perB);
    let anos = 0;
    while(true){
        A += parseInt(A*perA/100.0);
        B += parseInt(B*perB/100.0);
        anos += 1;
        if(anos > 100){
            console.log('Mais de 1 seculo.');
            break;
        }
        if(A > B){
            console.log(`${anos} anos.`);
            break;
        }
    }
}
